assets/js/utils/channelUtils.ts

```diff
@@ -135,7 +134,9 @@ export const syncPresenters = (
             email: email,
           };
       });
-      setPresenters(updatePresentersMicAndCamStatuses(presenters));
+      setPresenters(presenters);
+
+      // setPresentersStreams(prev => updatePresentersMicAndCamStatuses(prev));
```

Comment:
We want to rely only on WebRTC when it comes to everything, that happens inside `PresentersArea`, so if we need to update mic and cam statuses, this will be triggered by an event coming from webrtc.